Lib/xml/etree/ElementTree.py

```diff
@@ -1200,6 +1200,13 @@ def parse(source, parser=None):
     Return an ElementTree instance.
 
     """
+    if (getattr(source, 'encoding', None) == 'ISO-8859-1' and
+        source.mode == 'r'):
```

Comment:
IMO, this is a case where 80 chars is ok.

```suggestion
    if getattr(source, 'encoding', None) == 'ISO-8859-1' and source.mode == 'r':
```